src/common/catalog/catalog-entity.ts

```diff
@@ -230,7 +231,21 @@ export abstract class CatalogEntity<
     return this.metadata.name;
   }
 
+  /**
+	 * Trigger when user click on the icon in details panel
```

Comment:
not sure why github renders like this, this looks normal in my VSCode.

Reply:
Are you using tab characters?

Reply:
Thanks, fixed.